platforms/corda-ent/charts/idman/values.yaml

```diff
@@ -71,84 +61,117 @@ vault:
-  # Ex. port: 5052
+  # Eg. port: 5052
   port: 
 
+# Certificate Revocation Authority (CRA) service, inside the K8s cluster
 serviceRevocation:
-  # Revocation port inside cluster
-  # Ex port: 5053
+  # Eg. port: 5053
   port: 
 
-# ssh service related configuration
+# Provide the service details for setting up Shell access to the service.
 shell:
-  # Ex. sshdPort: 2222
+  # Te port to use for the SSH service
+  # Eg. sshdPort: 2222
   sshdPort: 
-  # ssh username
-  # Ex. user: idman
+  # SSH username
+  # Eg. user: idman
   user: 
-  # ssh password
-  # Ex. idmanP
+  # SSH password
+  # Eg. password:"idmanP
   password:
 
+#############################################################
+#                Database Options and Configuration         #
+#############################################################
 # Database configuration
 database:
-  # Ex. driverClassName: "org.h2.Driver"
+  # Java class name to use for the database
+  # Eg. driverClassName: "org.h2.Driver"
   driverClassName:
-  # Ex. url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
+  # The DB connection URL 
+  # Eg. url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
   url: 
-  # Ex. user: "example-db-user"
-  user: 
-  # Ex. password: "example-db-password"
+  # DB user
+  # Eg. user: "example-db-user"
+  user:
+  # DB password 
+  # Eg. password: "example-db-password"
   password: 
-  # Ex. runMigration: "true"
+  # Migrations of database can be run as part of the startup of Idman, if set to true. 
+  # If set to false, it will be run prior to setting up the Idman.
+  # Eg. runMigration: "true"
   runMigration: 
 
-# Specify the maximum size of the memory allocation pool
-# cordaJarMx: 1
-cordaJarMx: 
-
-# healthCheckNodePort set to get rid of logs pollution
-# Ex. healthCheckNodePort: 0
-healthCheckNodePort: 
+#############################################################
+#                       Settings                            #
+#############################################################
+# Provide volume related specifications
+volume:
+  # Eg. baseDir: /opt/corda
+  baseDir: 
 
-# where is CENM service jar file stored
-# Ex. bin
+# Provide the path where the CENM Idman .jar-file is stored
+# Eg. jarPath: bin
 jarPath: 
 
-# where is CENM service configuration file stored
-# Ex. etc
+# Provide the path where the CENM Service configuration files are stored
+# Eg. configPath: etc
 configPath: 
 
-# Sleep time after any error encountered in starting idman
-# Ex. 120
+# Provide any extra annotations for the PVCs
+pvc:
+  #   annotations:
+  #     key: "value"
+  annotations: {}
+
+# Provide any extra annotations for the deployment
+deployment:
+  #   annotations:
+  #     key: "value"
+  annotations: {}
+
+# Specify the maximum size of the memory allocation pool
+# Eg. cordaJarMx: 1
+cordaJarMx: 
```

Comment:
Memory allocation size should declare GB/MB here. This is hardcoded as G in deployment which is causing the pods to request a lot more memory than actually needed. Configuration should be available here, and hardcoded from tpl file.

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5):

```
# Provide configuration of the .jar files used in the Node
cordaJar:
  memorySize:
  unit:
  resources:
    limits:
    requests:
```
Corda .jar is dynamic for deployment, configurable in the `idman.tpl`